orttraining/orttraining/training_api/module.cc

```diff
@@ -132,9 +132,9 @@ Status Module::TrainStep(const std::vector<OrtValue>& inputs, std::vector<OrtVal
   feeds.insert(feeds.end(), weights_.begin(), weights_.end());
   feeds.insert(feeds.end(), gradients_.begin(), gradients_.end());
   // TODO: consider maintaining this as ortvalue instead of bool
```

Comment:
if it is easy to implement the TODO, shall we do it in this PR?

Reply:
the question is if we should do it, is it better to hold an ortvalue and update underlying buffer by unwrapping it every step, or to wrap it every step

Reply:
OK, I did not realize this. How about hold two ORTValues directly., one is True, one is False.  Never mind, let's refine it later.